client.go

```diff
@@ -84,7 +84,7 @@ func (myClient *Client) ParseCommand(firstLine []byte, isMultiplexing bool) (res
 		protocol.Debug("Received error from GetCommand: %s\r\n", err)
 		return false, err
 	}
-//	protocol.Debug("Received %s %s\r\n", myClient.command[0:myClient.commandLength], myClient.firstArgument)
+	protocol.Debug("Received %s %s\r\n", myClient.command[0:myClient.commandLength], myClient.firstArgument)
```

Comment:
Without the dev flag this should end up doing nothing (https://github.com/forcedotcom/rmux/blob/d28bbcda5191004c03e023f875296eee1fb2d18a/protocol/prod_debug.go#L21-21), but I should disassemble the binary and check to see if the function call is optimized out or not.